.gitignore

```diff
@@ -0,0 +1,6 @@
+__pycache__
+mvp-0/kubernetes-monitoring
+mvp-0/kaas/*.db
+mvp-0/kaas/.venv
+mvp-0/kaas/manifests/observer-kubeconfig.yaml
```

Comment:
why is this excluded? is this a temporary artifact of the development?

Reply:
It is a file that will be created when you follow the README tutorial on how to deploy "Monitoring of the KaaS layer use case".  see https://github.com/SovereignCloudStack/k8s-observability/pull/19/files#diff-b335630551682c19a781afebcf4d07bf978fb1f8ac04c6bf87428ed5106870f5R162